### ISSUE TITLE
Update broken UNIX link and check other resource links

### ISSUE THREAD

**Issue by Jen-OMalley** [AlexsLemonade/reproducible-research#144]
The `UNIX Tutorial for Beginners` link, which is the last link on [this page](https://alexslemonade.github.io/reproducible-research/reproducibility_resources.html), is no longer working. We can replace it with this one. http://linuxclass.heinz.cmu.edu/doc/Unix-Tutorial-surrey/

May as well double check all the other links on this page too!

**Comment by sjspielman** [comment]
> We can replace it with this one. http://linuxclass.heinz.cmu.edu/doc/Unix-Tutorial-surrey/

Seems like a good replacement, but OTOH I don't know if it will also go down. Seems to be associated with a particular class this professor teaches. A lot of other websites have copied the original tutorial over too (e.g., https://users.cs.duke.edu/~alvy/courses/unixtut/), and in general who knows how stable any of them are...! So, I've just convinced myself that eh, we'll do the best we can, since we aren't going to put the effort into making our own version. Any replacement is ok.

**Comment by Jen-OMalley** [comment]
True. We opted to use that link as a replacement here https://www.ccdatalab.org/research-resources#unix. 

I periodically check that all links are working on our website, so I'll keep an eye on it!
